docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+
+ENV GOROOT=/usr/local/go
+
+RUN go get github.com/tools/godep
+RUN go get github.com/hudl/fargo
```

Comment:
@seh: I haven't been following all of this, but I'd tend to agree. Another scenario: if we want to set this up to automatically run tests through Docker somewhere, it seems logical for it to use the latest source for each branch we're testing.